libs/src/services/ethWeb3Manager/index.js

```diff
@@ -15,7 +15,19 @@ class EthWeb3Manager {
     if (!web3Config.ownerWallet) throw new Error('missing web3Config property: ownerWallet')
 
     // Pick a provider at random to spread the load
-    const provider = sample(web3Config.providers)
+    const providers = web3Config.providers.map(provider => {
+      if (typeof provider === 'string') {
+        if (provider.startsWith('http')) {
+          return new Web3.providers.HttpProvider(provider)
+        } else {
+          return new Web3.providers.WebsocketProvider(provider)
+        }
+      } else {
+        return provider.eth.currentProvider
+      }
+    })
+
+    const provider = new MultiProvider(providers)
```

Comment:
subtle behavioral difference here is that a new provider could be used for each contract interaction. this will also impact audius-client, but i think this change should be fine. we also only use 1 provider on the client for now, which is just eth.audius.co